<ui version="4.0" >
 <class>DicToolsWidget</class>
 <widget class="QWidget" name="DicToolsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>_("Dictionary tools")</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCheck" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>440</width>
        <height>308</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>_("Check")</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QLineEdit" name="lineEditCheck" />
       </item>
       <item>
        <widget class="QLabel" name="labelCheck" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlus1" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>440</width>
        <height>308</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>_("Plus 1")</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QLineEdit" name="lineEditPlus1" />
       </item>
       <item>
        <widget class="QTreeView" name="treeViewPlus1" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRegexp" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>440</width>
        <height>308</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>_("Regular expressions")</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLineEdit" name="lineEditRegexp" />
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>_("Minimum length:")</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxMinLength" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>_("Maximum length:")</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxMaxLength" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>15</number>
           </property>
           <property name="value" >
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewRegexp" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLimitReached" >
         <property name="text" >
          <string>_("Note: only the 1000 first results have been displayed")</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
